README.md

```
@@ -1 +1,29 @@
+Config
+
+Create services to move the logic away from controllers:
+
+- A declared type for sending data objects to services.
+- The same type will be used by Services to return the data back to the controllers.
+
+
+Abstract away ORM model and create a physical store like APIs using repository pattern with interface for stores and their concrete implementation utilizing ORM models:
+
+- Repository pattern is an added layer of abstraction that separates out data access from business logic.
+- 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)